src/Symfony/Component/Translation/PseudoLocalizationTranslator.php

```diff
@@ -0,0 +1,273 @@
+                $visibleText .= $text;
+            }
+
+            if (!$localizable) {
+                $trans .= $text;
+
+                continue;
+            }
+
+            $trans .= $this->accents ? strtr($text, [
+                ' ' => ' ',
+                '!' => '¡',
+                '"' => '″',
+                '#' => '♯',
+                '$' => '€',
+                '%' => '‰',
+                '&' => '⅋',
+                '\'' => '´',
+                '(' => '{',
+                ')' => '}',
+                '*' => '⁎',
+                '+' => '⁺',
+                ',' => '،',
+                '-' => '‐',
+                '.' => '·',
+                '/' => '⁄',
+                '0' => '⓪',
+                '1' => '①',
+                '2' => '②',
+                '3' => '③',
+                '4' => '④',
+                '5' => '⑤',
+                '6' => '⑥',
+                '7' => '⑦',
+                '8' => '⑧',
+                '9' => '⑨',
+                ':' => '∶',
+                ';' => '⁏',
+                '<' => '≤',
+                '=' => '≂',
+                '>' => '≥',
+                '?' => '¿',
+                '@' => '՞',
+                'A' => 'Å',
+                'B' => 'Ɓ',
+                'C' => 'Ç',
+                'D' => 'Ð',
+                'E' => 'É',
+                'F' => 'Ƒ',
+                'G' => 'Ĝ',
+                'H' => 'Ĥ',
+                'I' => 'Î',
+                'J' => 'Ĵ',
+                'K' => 'Ķ',
+                'L' => 'Ļ',
+                'M' => 'Ṁ',
+                'N' => 'Ñ',
+                'O' => 'Ö',
+                'P' => 'Þ',
+                'Q' => 'Ǫ',
+                'R' => 'Ŕ',
+                'S' => 'Š',
+                'T' => 'Ţ',
+                'U' => 'Û',
+                'V' => 'Ṽ',
+                'W' => 'Ŵ',
+                'X' => 'Ẋ',
+                'Y' => 'Ý',
+                'Z' => 'Ž',
+                '[' => '⁅',
+                '\\' => '∖',
+                ']' => '⁆',
+                '^' => '˄',
+                '_' => '‿',
+                '`' => '‵',
+                'a' => 'å',
+                'b' => 'ƀ',
+                'c' => 'ç',
+                'd' => 'ð',
+                'e' => 'é',
+                'f' => 'ƒ',
+                'g' => 'ĝ',
+                'h' => 'ĥ',
+                'i' => 'î',
+                'j' => 'ĵ',
+                'k' => 'ķ',
+                'l' => 'ļ',
+                'm' => 'ɱ',
+                'n' => 'ñ',
+                'o' => 'ö',
+                'p' => 'þ',
+                'q' => 'ǫ',
+                'r' => 'ŕ',
+                's' => 'š',
+                't' => 'ţ',
+                'u' => 'û',
+                'v' => 'ṽ',
+                'w' => 'ŵ',
+                'x' => 'ẋ',
+                'y' => 'ý',
+                'z' => 'ž',
+                '{' => '(',
+                '|' => '¦',
+                '}' => ')',
+                '~' => '˞', ]) : $text;
```

Comment:
the `]` of the end of the array should be on next line. In a multiline array, the closing bracket is not on the same line than the last item.